library/src/test/kotlin/com/chuckerteam/chucker/internal/data/repository/HttpTransactionDatabaseRepositoryTest.kt

```diff
@@ -155,13 +155,20 @@ internal class HttpTransactionDatabaseRepositoryTest {
             createRequest("def").withResponseData().apply {
                 requestDate = 300L
             }
+        val transactionFour =
```

Comment:
This should be in a separate test:

fun `transaction tuples are filtered by qraphQlOperationName`